packages/core/js-sdk/src/client.ts

```diff
@@ -1,5 +1,13 @@
 import qs from "qs"
-import { ClientFetch, Config, FetchArgs, FetchInput, Logger } from "./types"
+import { events } from "fetch-event-stream"
```

Comment:
comment(non-blocking): looks like this package is only a single file, so we could include it ourselves, if we don't want another dep

Reply:
Yeah it is, I also thought about copying it over, but I don't see the harm in having it as a dependency TBH

Reply:
I'll get this merged as I want to use it in platform already but if you think we should remove the dependency I'll do it in a separate PR